src/has.ts

```diff
@@ -0,0 +1,6 @@
+import global from 'dojo-core/global';
+import { add } from 'dojo-core/has';
+
+add('history', 'history' in global);
```

Comment:
+1 to `html5-history`.  Yeah I guess the style guide could use a couple of updates.